lib/src/main/java/com/basistheory/android/view/TextElement.kt

```diff
@@ -135,37 +139,70 @@ open class TextElement : FrameLayout {
         subscribeToInputEvents()
     }
 
-    private fun addMask(mask: List<Any>) {
-        maskWatcher = MaskWatcher(mask)
-        input.addTextChangedListener(maskWatcher)
-    }
-
-    private fun removeMask() {
-        input.removeTextChangedListener(maskWatcher)
-        maskWatcher = null
-    }
+    protected open fun transformUserInput(userInput: String?): String? = userInput
```

Comment:
why is this needed if we have transform above?

Reply:
This is an internal transform that an element can use to transform user input. The other transform is publicly exposed and is used to transform before tokenization - I can't change that name so that it matches the web/ios interfaces.